### PR TITLE
[15.0][MIG] pos_user_restriction: Migration to 15.0

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: pos_user_restriction/models/pos_session.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    def set_cashbox_pos(self, cashbox_value, notes):
+        if self.env.user.has_group(
+            "pos_user_restriction.group_assigned_points_of_sale_user"
+        ):
+            res = super(
+                PosSession, self.with_context(bypass_pos_user=True)
+            ).set_cashbox_pos(cashbox_value, notes)
+        else:
+            res = super().set_cashbox_pos(cashbox_value, notes)
+        return res
```
Comment (yankinmax) [inline pos_user_restriction/models/pos_session.py]:
```
if self.env.user.has_group(
    "pos_user_restriction.group_assigned_points_of_sale_user"
):
    self = self.with_context(bypass_pos_user=True)
return super().set_cashbox_pos(cashbox_value, notes)
```

File: pos_user_restriction/models/pos_session.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    def get_closing_control_data(self):
+        if self.env.user.has_group(
```
Comment (yankinmax) [inline pos_user_restriction/models/pos_session.py:15]: same here

**Review by yankinmax — APPROVED** [review]
Did you check it functionally?

**Comment by ghost** [comment]
@yankinmax yes !

**Review by Camille0907 — APPROVED** [review]
Technically looks good, thanks :)

**Pull request merged**